src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/ControllerTagTest.kt

```diff
@@ -0,0 +1,50 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource
+
+import io.swagger.v3.oas.annotations.tags.Tag
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.fail
+import org.springframework.context.annotation.ClassPathScanningCandidateComponentProvider
+import org.springframework.core.type.filter.AnnotationTypeFilter
+import org.springframework.web.bind.annotation.RestController
+import java.lang.reflect.AnnotatedElement
+import java.lang.reflect.Method
+
+class ControllerTagTest {
```

Comment:
A test that will fail if we add any new endpoints but forget to tag the controller or endpoint